resource-managers/yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnSchedulerBackend.scala

```diff
@@ -306,8 +308,8 @@ private[spark] abstract class YarnSchedulerBackend(
             .recover {
               case NonFatal(e) =>
                 logWarning(log"Attempted to get executor loss reason for executor id " +
-                  log"${MDC(EXECUTOR_ID, executorId)} at RPC address " +
-                  log"${MDC(HOST_PORT, executorRpcAddress)}, but got no response. " +
+                  log"${MDC(LogKey.EXECUTOR_ID, executorId)} at RPC address " +
```

Comment:
If imported in the form of `import org.apache.spark.internal.LogKey.{EXECUTOR_ID, HOST_PORT, REASON ...}` in the header, it will be very `long`, so it is directly imported as `import org.apache.spark.internal.LogKey` here and prefixed in the code.